docs/simulation/dynamic/index.md

```diff
@@ -24,18 +24,18 @@ The inputs of a dynamic simulation are the following:
 
 (dynamic-models-mapping)=
 ### Dynamic models mapping
-For the moment, the only way to associate dynamic models to static components is through a groovy script. Note that the syntax of this script is specific to each simulator:
-- [Dynawo dynamic model DSL](inv:powsybldynawo:*:*#dynamic_simulation/dynamic-models-dsl)
+The dynamic models may be provided through a groovy script thanks to the `GroovyDynamicModelsSupplier` provided in powsybl-dynamic-simulation-dsl artifact. Note that the syntax of this groovy script is specific to each simulator.
+See [Dynawo dynamic model mapping](inv:powsybldynawo:*:*#dynamic_simulation/dynamic-models-mapping) for Dynawo specific DSL and others mapping methods. 
 
 (event-models-mapping)=
 ### Event models mapping
-For the moment, the only way to add events to the simulation is through a groovy script. Note that the syntax of this script is specific to each simulator:
-- [Dynawo event model DSL](inv:powsybldynawo:*:*#dynamic_simulation/event-models-dsl)
+The event models may be provided through a groovy script thanks to the `GroovyEventModelsSupplier` provided in powsybl-dynamic-simulation-dsl artifact. Note that the syntax of this groovy script is specific to each simulator.
+See [Dynawo event model DSL](inv:powsybldynawo:*:*#dynamic_simulation/event-models-mapping) for Dynawo specific DSL and others mapping methods.
```

Comment:
As mentioned in the PR powsybl/powsybl-dynawo#410, could we speak of mapping for events? and  for output variables?